__tests__/data/hs_router_output.json

```diff
@@ -4,95 +4,58 @@
       "version": "1",
       "type": "REST",
       "method": "POST",
-      "endpoint": "https://api.hubapi.com/contacts/v1/contact/batch/",
+      "endpoint": "https://api.hubapi.com/crm/v3/objects/contacts/batch/create",
```

Comment:
Why existing test cases are mutated, we are keeping backward compatibility right?

Reply:
This is for rETL changes. Here mappedToDestination is set to true